contrib/check_go_compiler_version.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+#
+# Copyright (C) 2021, NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# See file LICENSE for terms.
+#
+
+function ver { printf "%03d%03d%03d%03d" $(echo "$1" | tr '.' ' '); }
+
+goversion=`go version | awk '{print substr($3, 3, length($3)-2)}'`
+
+[ $(ver $goversion) -ge $(ver $1) ] && exit 0
```

Comment:
maybe use m4_version_compare, so need another script file and put all inside go.m4?